### ISSUE TITLE
Test images with provided ILSVRC2016 trained model 

### ISSUE THREAD

**Issue by SJTUzhanglj** [weiliu89/caffe#377]
I want to test with the provided ilsvrc2016 model on my own images, is it necessary to download the huge ilsvrc2016 dataset and generate the lmdb and 'val2_name_size' file, according to the README?

**Comment by weiliu89** [comment]
No. You can remove that in the deploy.prototxt if it gives any error.
